### PR TITLE
Initial Project Setup

### PR DESCRIPTION
**Pull request by Vylpes**
- Installs the initial node packages
- Setups the project layout
- Adds alerts
- Adds initial base string

### CODE REVIEW

File: sonar.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    res.locals.message = err.message;
+    res.locals.error = req.app.get('env') === 'development' ? err : {};
+
+    // render the error page
+    res.status(err.status || 500);
+    res.render('error');
+});
+
+app.listen('3000', () => {
+    console.log('app listening on port 3000');
```
[the remark this answers is not in the document]
Comment (Vylpes) [reply sonar.js:39]: Might change this rule

File: sonar.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'pug');
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
```
[the remark this answers is not in the document]
Comment (Vylpes) [reply sonar.js:17]: Need to add to the config

**Comment by Vylpes** [comment]
- [ ] Add unit tests

File: helpers/alerts.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+exports.info = (str, message) => {
```
Comment (VylpesTester) [inline helpers/alerts.js:1]: Would it be better to make the alerts a class? Such as:
- Create the class, which fetches the base string
- Have the alerts as methods under the class
- When being fetched for the pug you have another method which gets the final string

File: .gitignore [hunk 4]
```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Exceptions
+!/public/dist
```
Comment (VylpesTester) [inline .gitignore:107]: Add an ignore for .DS_Store

**Review by VylpesTester — CHANGES_REQUESTED** [review]

**Comment by Vylpes** [comment]
- [ ] Convert to TypeScript
- [ ] Setup TSLint
- [ ] Setup GitHub actions

**Review by VylpesTester — APPROVED** [review]

**Pull request merged**